data_write/data_write.py

```diff
@@ -52,67 +51,69 @@ def printing(msg):
-    "int_time" : None, # Integration time in seconds.
-    "tx_pulse_len" : None, # Length of the pulse in microseconds.
-    "tau_spacing" : None, # The minimum spacing between pulses in microseconds.
-                          # Spacing between pulses is always a multiple of this.
-    "main_antenna_count" : None, # Number of main array antennas.
-    "intf_antenna_count" : None, # Number of interferometer array antennas.
-    "freq" : None, # The frequency used for this experiment slice in kHz.
-    #"filtered_3db_bandwidth" : None, # Bandwidth of the output iq data types? can add later
-    "rx_center_freq" : None, # the center frequency of this data (for rawrf), kHz
-    "samples_data_type" : None, # C data type of the samples such as complex float.
-    "pulses" : None, # The pulse sequence in units of the tau_spacing.
-    "pulse_phase_offset" : None, # For pulse encoding phase. Contains one phase offset per pulse in pulses.
-    "lags" : None, # The lags created from two pulses in the pulses array.
-    "blanked_samples" : None, # Samples that have been blanked because they occurred during transmission times.
-                              # Can differ from the pulses array due to multiple slices in a single sequence.
-    "sqn_timestamps" : None, # A list of GPS timestamps of the beginning of transmission for each
-                             # sampling period in the integration time. Seconds since epoch.
-    "beam_nums" : None, # A list of beam numbers used in this slice.
-    "beam_azms" : None, # A list of the beams azimuths for each beam in degrees off boresite.
-    "noise_at_freq" : None, # Noise at the receive frequency, should be an array (one value per sequence) (TODO units??) (TODO document FFT resolution bandwidth for this value, should be = output_sample rate?)
-    #"noise_in_raw_band" : None, # Average noise in the sampling band (input sample rate) (TODO units??)
-    #"rx_bandwidth" : None, # if the noise_in_raw_band is provided, the rx_bandwidth should be provided!
-    "num_samps" : None, # Number of samples in the sampling period.
-    "antenna_arrays_order" : None, # States what order the data is in. Describes the data layout.
-    "data_descriptors" : None, # Denotes what each data dimension represents.
-    "data_dimensions" : None, # The dimensions in which to reshape the data.
-    "data_normalization_factor" : None, # The scale of all of the filters, multiplied, for a total scaling factor to normalize by.
-    "data" : [], # A contiguous set of samples (complex float) at given sample rate
-    "correlation_descriptors" : None, # Denotes what each acf/xcf dimension represents.
-    "correlation_dimensions" : None, # The dimensions in which to reshape the acf/xcf data.
-    "main_acfs" : [], # Main array autocorrelations
-    "intf_acfs" : [], # Interferometer array autocorrelations
-    "xcfs" : [] # Crosscorrelations between main and interferometer arrays
```

Comment:
Note that the only changes here are formatting changes (two spaces before a comment, no space before a colon, etc)